File: app/src/main/java/pl/mobilization/konfeocheckin/KonfeoEndpoint.java
package pl.mobilization.konfeocheckin;

import okhttp3.ResponseBody;
import retrofit2.Call;
import retrofit2.http.GET;

/**
 * Created by marekdef on 26.12.17.
 */

public interface KonfeoEndpoint {

    @GET("/en/login")
    Call<ResponseBody> login();
}

File: app/src/main/java/pl/mobilization/konfeocheckin/KonfeoEndpoint.kt
package pl.mobilization.konfeocheckin

import org.jsoup.nodes.Document

import okhttp3.ResponseBody
import retrofit2.Call
import retrofit2.http.GET

/**
 * Created by marekdef on 26.12.17.
 */

interface KonfeoEndpoint {

    @GET("/en/login")
    fun login(): Call<Document>

    @GET("/en/login")
    fun login2(): Call<LoginPage>
}
